rules-reviewed/eap7/eap6/resteasy.windup.xml

```diff
@@ -0,0 +1,826 @@
+                    <tag>resteasy</tag>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="resteasy-eap6-000021">
+            <when>
+                <javaclass references="org.jboss.resteasy.spi.Link" />
+            </when>
+            <perform>
+                <hint title="Deprecated Link class" effort="1" severity="mandatory">
+                    <message>Replaced by `javax.ws.rs.core.Link` in JAX-RS API.</message>
+                    <link title="RESTeasy SPI Application Changes"
+                        href="https://access.qa.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/version-7.0/migration-guide/#migrate_resteasy_deprecated_classes" />
+                    <link title="JAX-RS API Link" href="https://docs.oracle.com/javaee/7/api/index.html?javax/ws/rs/core/Link.html" />
+                    <tag>resteasy</tag>
+                    <tag>jax-rs</tag>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="resteasy-eap6-000022">
+            <when>
+                <javaclass references="org.jboss.resteasy.spi.interception.ClientExecutionContext" />
+            </when>
+            <perform>
+                <hint title="Deprecated interface ClientExecutionContext" effort="1" severity="mandatory">
+                    <message>Use ### instead of `org.jboss.resteasy.spi.interception.ClientExecutionContext`.
+                    The Resteasy interceptor facility introduced in release 2.x is replaced by the JAX-RS 2.0 compliant interceptor facility in release 3.0.x.</message>
+                    <link title="RESTeasy SPI Application Changes"
+                        href="https://access.qa.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/version-7.0/migration-guide/#migrate_resteasy_deprecated_classes" />
+                    <tag>resteasy</tag>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="resteasy-eap6-000023">
+            <when>
+                <javaclass references="org.jboss.resteasy.spi.interception.ClientExecutionInterceptor" />
+            </when>
+            <perform>
+                <hint title="Deprecated ClientExecutionInterceptor interface" effort="1" severity="mandatory">
+                    <message>Use ### instead of `org.jboss.resteasy.spi.interception.ClientExecutionInterceptor`.
+                    The Resteasy interceptor facility introduced in release 2.x is replaced by the JAX-RS 2.0 compliant interceptor facility in release 3.0.x.</message>
+                    <link title="RESTeasy SPI Application Changes"
+                        href="https://access.qa.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/version-7.0/migration-guide/#migrate_resteasy_deprecated_classes" />
+                    <tag>resteasy</tag>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="resteasy-eap6-000024">
```

Comment:
Duplicates `resteasy-eap6-000015` and uses wrong FQN in description. Should IMHO be deleted.